src/components/DataCollector/index.js

```diff
@@ -51,7 +52,8 @@ createDataCollector.namespace = "DataCollector";
 
 createDataCollector.configValidators = {
   collectionUrl: {
-    defaultValue: "https://edgegateway.azurewebsites.net"
+    defaultValue: "https://edgegateway.azurewebsites.net",
+    validate: [required, matchesRegEx("^https://[^s/$.?#].[^s]*$")]
```

Comment:
Also, did you mean to escape the s in your regex to match spaces?  and should it be a case insensitive match?

Reply:
Looks like it was automatically stripped of some characters during commit